.travis.sh

```diff
@@ -57,5 +57,5 @@ elif [[ ${MODE} = cmake-oos ]]; then
     make DESTDIR="${PWD}"/ROOT install
     find ROOT -printf "%P\n" | sort
 else
-    ./qa.sh
+    ./qa.sh ${CONFIGURE_ARGS}
```

Comment:
It looks fair to me that this can be in another PR as it is not directly related to Fuzzit, but rather needed by Fuzzit as part of the continuous integration